README.md

```diff
@@ -335,9 +335,30 @@ to the configuration. The EVCC loadpoints can then be added to the SHM configura
 Sunny-Portal via the "Optional energy demand" slider. When the amount of configured PV is not available, charging suspends like in **PV** mode. So, pushing the slider completely
 to the left makes **Min+PV** behave as described above. Pushing completely to the right makes **Min+PV** mode behave like **PV** mode.
 
+### Energy Tariffs & Savings Estimate
+
+You can specify your energy tariffs and grid feed-in rates. Evcc uses these values for a rough savings calculation shown in the web UI.
+
+```yaml
+tariffs:
+  grid:
+    # static grid price
+    type: fixed
+    price: 29 # ct/kWh
```

Comment:
Macht es Sinn hier ein Beispiel mit nicht ganzen Zahlen zu nehmen um, das Format klar zu machen?
```suggestion
    price: 29.9 # ct/kWh
```

Reply:
Preise werden nun in der Währungshaupteinheit angegeben (EUR anstatt cent). Dadurch wird's auch in der Doku klarer.